CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1 (Unreleased)
```

Comment:
Is this current release that is lining up with Boundary 0.1.5 going to be 1.0.1 of the provider? If so, should we add a new unreleased section and update the date for 1.0.1?